SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The versions of the project that are currently supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.x | :white_check_mark: |
+| 3.x | :white_check_mark: |
+| < 3.0 | :x: |
+
+## Reporting a Vulnerability
+
+You can report a vulnerability following the instructions in https://tidelift.com/docs/security.
```

Comment:
So they're reported to tidelift and tidelift reports it to whom?

Reply:
Tidelift reports it to us, after filtering the noise. What alternative do we have? Do you prefer that we use our official email? `material.ui.org@gmail.com`

Reply:
> Tidelift reports it to us 

But how do they do that if not by email? I can't imagine Tidelift filtering issues for us for free.

I'll have to rewatch the demo but I thought GitHub had a report mechanism built-in.

Reply:
I can't see anything about it. Let's start simple: material.ui.org@gmail.com.